docs/UserGuide.md

```diff
@@ -390,7 +391,7 @@ Examples:
 * `edit 1 p/91234567`, assuming there is a contact to edit, followed by `undo` will revert the edit of the contact.
 * Assuming there are 3 contacts, `delete 1`, which deletes the first contact, followed by `clear`, which clears the
   remaining 2 contacts, followed by `undo` will only add the 2 cleared contacts back. A subsequent `undo` will add
-  back the contact deleted.
+  back the contact deleted at the beginning.
```

Comment:
I believe it should be "add _something_ back" instead of "add back _something_"

Reply:
also, I think the full-stop at the end should be removed as well

Reply:
I see, updated! Thanks!